tests/builds_test.go

```diff
@@ -53,25 +55,27 @@ var _ = Describe("Builds", func() {
 
 		Context("with no app", func() {
 			It("cannot create a build without existing app", func() {
-				cmd, err := start("deis builds:create %s -a %s", exampleImage, testApp.Name)
+				sess, err := start("deis builds:create %s -a %s", testData.Profile, exampleImage, testApp.Name)
+				Eventually(sess.Err).Should(Say("404 Not Found"))
+				Eventually(sess).Should(Exit(1))
 				Expect(err).NotTo(HaveOccurred())
-				Eventually(cmd.Err).Should(Say("404 Not Found"))
-				Eventually(cmd).Should(Exit(1))
 			})
 		})
 
 		Context("with existing app", func() {
+			var testData TestData
 
 			BeforeEach(func() {
-				createApp(testApp.Name)
-				createBuild(exampleImage, testApp)
+				testData = initTestData()
+				createApp(testData.Profile, testApp.Name)
+				createBuild(testData.Profile, exampleImage, testApp)
 			})
 
 			It("can list app builds", func() {
-				cmd, err := start("deis builds:list -a %s", testApp.Name)
+				sess, err := start("deis builds:list -a %s", testData.Profile, testApp.Name)
+				Eventually(sess).Should(Say(uuidRegExp))
+				Eventually(sess).Should(Exit(0))
 				Expect(err).NotTo(HaveOccurred())
-				Eventually(cmd).Should(Exit(0))
-				Eventually(cmd).Should(Say(uuidRegExp))
```

Comment:
Ah, ok. Maybe related to waiting for the buffer to fill up. Thanks for explaining, and sgtm